packages/react-native/ios/Logger.swift

```diff
@@ -29,6 +29,16 @@ import Capture
         }
     }
 
+    @objc
+    public static func getDeviceID(_ resolve: @escaping RCTPromiseResolveBlock, rejecter reject: @escaping RCTPromiseRejectBlock) {
+        let deviceID = Capture.Logger.deviceID
+        if !deviceID.isEmpty {
+            resolve(deviceID)
+        } else {
+            resolve("Unknown")
+        }
+    }
```

Comment:
Is there a reason to use a promise interface here when there isn't anything async going on?

Reply:
Quite probably that's unnecessary. While I was attempting to get things wired up for Android, when I didn't use the promise interface it only ever seemed to return `undefined`. Could be that I did something else to fix it ultimately. I'll try changing back and see if it breaks.